### PR TITLE
Add merge command

### PR DESCRIPTION
**Pull request by Porama6400**
- Add merge command
- Add merge command to readme
- Add help command to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Id   | Name
 ## Console Commands
 The current available console commands are listed here. Command names are not case sensitive, but player names are.
 
+ - Help
```
Comment (Porama6400) [inline README.md]: it previous command that i make
but i forgot to add readme

**Comment by DaFudgeWizzad** [comment]
Dont forget to add to the help command in commandList.js

File: src/modules/CommandList.js [hunk 2]
```diff
@@ -26,8 +26,31 @@ var fillChar = function (data, char, fieldLength, rTL) {
...
+                }
+
+                console.log("[Console] Forced " + client.name + " to merge cells");
+                break;
+            }
+        }
+    },
     help: function(gameServer,split) {
         console.log("[Console] ======================== HELP ======================");
+        console.log("[Console] merge      : force player to merge cells");
```
Comment (Porama6400) [inline src/modules/CommandList.js]: Add Merge command to Help

**Comment by Porama6400** [comment]
thanks

**Comment by DaFudgeWizzad** [comment]
No problem

**Comment by JCookie20001** [comment]
Don't you have to reset the recombineTicks back to the default number?

**Comment by DerpyUnlimited** [comment]
@Luka967 where would you put that line of code in?

**Comment by ItzLevvie** [comment]
Replace ```client.cells[j].calcMergeTime(0);``` with ```client.cells[j].recombineTicks = 0;```

**Comment by ItzLevvie** [comment]
Change ```client.cells[j].calcMergeTime(0);``` to ```client.cells[j].recombineTicks = 0;```

**Comment by Luka967** [comment]
@ItzLevvie It's not possible anymore as my merged pull request on revamping merging ticks from 0 to infinity. However `client.cells[j].calcMergeTime(0);` will work as intended (i'm too lazy to tell you why. Take a look at the function).

**Comment by DerpyUnlimited** [comment]
@ItzLevvie or @Luka967 where is that line of code??

**Pull request closed without merging**